web/client/components/styleeditor/Editor.jsx

```diff
@@ -236,7 +225,7 @@ class Editor extends React.Component {
                                 this.props.onChange(code);
                             }
                         }, this.props.waitTime);
-                        CM.extendMode(this.props.mode, { hintProperties: this.props.hintProperties });
+                        cm.extendMode(this.props.mode, { hintProperties: this.props.hintProperties });
```

Comment:
this.cm is better, right?
```suggestion
                        this.cm.extendMode(this.props.mode, { hintProperties: this.props.hintProperties });
```